### PR TITLE
Support nvdimm only on x86_64

### PR DESCRIPTION
**Pull request by rvykydal**
Resolves: rhbz#1602436

### CODE REVIEW

**Comment by rvykydal** [comment]
jenkins, test this please

**Comment by rvykydal** [comment]
jenkins, test this please

File: pyanaconda/kickstart.py [hunk 1]
```diff
@@ -1218,6 +1218,10 @@ class Nvdimm(commands.nvdimm.RHEL7_Nvdimm):
     def parse(self, args):
         action = commands.nvdimm.RHEL7_Nvdimm.parse(self, args)
 
+        if not blivet.nvdimm.nvdimm().nvdimm_plugin_available:
```
Comment (poncovka) [inline pyanaconda/kickstart.py]: I think that `blivet.nvdimm.nvdimm` is already the instance of NVDIM you want, so why do you call `nvdimm()`?

Comment (rvykydal) [reply pyanaconda/kickstart.py]: It is just consistent with already present nvdimm singleton calls, and also with how we are already using eg iscsi, fcoe and ZFCP singleton. Which I hesitate to change in scope of this PR.

Comment (poncovka) [reply pyanaconda/kickstart.py]: No problem, I was just curious.

**Comment by jkonecny12** [comment]
Just to make things more obvious this is blocked by https://github.com/storaged-project/blivet/pull/707 .

**Pull request merged**